src/eventListener.ts

```diff
@@ -1,19 +1,35 @@
-import { addColonyEventListener, addTokenEventListener } from './utils';
+import {
+  addColonyEventListener,
+  addNetworkEventListener,
+  addTokenEventListener,
+} from './utils';
 import { ContractEventsSignatures } from './types';
 
 /*
  * All events that need to be tracked on a single colony go in here
  * as this will get linked in all the relevant places
  */
-export const colonySpecificEventsListener = async (colonyAddress: string): Promise<void> => {
-  await addColonyEventListener(ContractEventsSignatures.ColonyFundsClaimed, colonyAddress);
+export const colonySpecificEventsListener = async (
+  colonyAddress: string,
+): Promise<void> => {
+  await addColonyEventListener(
+    ContractEventsSignatures.ColonyFundsClaimed,
+    colonyAddress,
+  );
   await addTokenEventListener(ContractEventsSignatures.Transfer, colonyAddress);
 };
 
+/*
+ * No general, global level (basically network) events to listen to currently
+ */
```

Comment:
Is this comment still relevant since we're now listening to the network events below?

Reply:
Haha, you're right, I don't think I read it fully before just moving it above the handler 😅